bin/storm.py

```diff
@@ -241,7 +240,6 @@ def jar(jarfile, klass, *args):
             extrajars=[tmpjar, USER_CONF_DIR, STORM_BIN_DIR],
             args=args,
             daemon=False,
```

Comment:
@hmcl I don't see a reason to fork the submission process.

I'm +1 for the patch.